### ISSUE TITLE
Template loading order

### ISSUE THREAD

**Issue by henhuy** [rl-institut/WAM#12]
Right now, templates are tried to be loaded in this order:

1. wam/templates/
1. wam/stemp/templates/
1. wam/stemp_abw/templates/
1. ... (all wam-apps)
1. django default templates
1. ...

Thus, a template in stemp_abw which has same name as a template in stemp will not be loaded, but the stemp template instead...
Possible solutions:
* Nest create "widgets" folder into app_name/templates/app_name/ folder or
* Create new look-up order, like:

Suggested loading order:
1. current app always comes first!
1. wam template
1. django default templates
1. ... and dont look at other apps!

**Comment by Bachibouzouk** [comment]
A solution for that could also be to add the name of the app in front the the template name,

**Comment by nesnoj** [comment]
Yes, this should work but will lead to long, bulky names such as `an_advanced_super_app-layer_region_select.html`. Furthermore, if app developers do not follow the naming rules (I admit it's quite unlikely that two devs are this nasty ;) we get back to the same problem..
